homeworks/roman.pavliuk_RomanPvlk/5-logic/homework.js

```diff
@@ -0,0 +1,32 @@
+function calculate(x, y, operation) {
+    let result = 0;
+    switch (operation) {
+        case '+':
+            result = x + y;
+            break;
+        case '-':
+            result = x - y;
+            break;
+        case '*':
+            result = x * y;
+            break;
+        case '/':
+            if (y === 0) {
+                return 'you can not divide by zero =(';
+            }
+            result = x / y;
+            break;
+        default:
+            return 'Choose a valid operation';
+    }
+    if (Number.isNaN(x) || Number.isNaN(y)) {
+        return 'Enter a number';
```

Comment:
Should be an error for empty field 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/4b25e3ce-3894-40ab-921b-903ecd439325)